src/shogun/optimization/ElasticNetPenalty.h

```diff
@@ -92,11 +102,20 @@ class ElasticNetPenalty: public SparsePenalty
 		return grad;
 	}
 
-	virtual void set_rounding_eplison(float64_t eplison)
+	/** Set the rounding epslion for L1 penalty
+	 *
+	 * @param epslion rounding epslion
+	 *
```

Comment:
I'd say let's try to be consistent with the doc in the other methods and remove this line.